src/index.js

```diff
@@ -9,28 +9,120 @@ function prependActionButton(button) {
   actionsList.prepend(button);
 }
 
+function prependEpicSelect(epics) {
+  const actionsList2 = document.querySelector(
+    ".window-module.other-actions > div"
+  );
+
+  var select = '<select id="mactrelly-jira-selector" style="width: 154px;height: 29px;overflow: hidden; background-color: #0079bf; color: white; border-radius: 5px; padding: 2px 10px; box-shadow: 1px 1px 11px #330033; ">';
+  for (var key in epics) {
+    select = select + (`<option value="${epics[key]}">${epics[key]} | ${key}</option>`)
+  }
+
+  select = select + '</select>]';
+
+  actionsList2.innerHTML = select + actionsList2.innerHTML;
+}
+
+function httpGet(theUrl)
+{
+    const xmlHttp = new XMLHttpRequest();
+    xmlHttp.open( "GET", theUrl, false ); // false for synchronous request
```

Comment:
can we remove the spaces at start and end to keep it consistent with the other ones